db/upgrade/rdb_modern/views/005-v_nrte_srte_tables.sql

```diff
@@ -0,0 +1,52 @@
+use rdb_modern;
```

Comment:
We don't specify the database in the scripts. This can be removed 👍

Reply:
@upasanapattnaik-eq  we need to do the "use" statement. otherwise it will get created in other database's that s defaulted. WE should not have it like <db>.dbo.<table>. IT is ok to have it at the top of the script.

Reply:
Makes sense! Aligned 🚀